pkg/build/registry/buildconfiginstantiate/rest.go

```diff
@@ -209,7 +210,7 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
 	if err != nil {
 		return nil, errors.NewInternalError(fmt.Errorf("unable to connect to server: %v", err))
 	}
-	if err := exec.Stream(r, nil, nil, false); err != nil {
+	if err := exec.Stream(kubeletremotecommand.SupportedStreamingProtocols, r, nil, nil, false); err != nil {
```

Comment:
It's backwards compatible

On Wednesday, April 20, 2016, Clayton Coleman <notifications@github.com>
wrote:

> In pkg/build/registry/buildconfiginstantiate/rest.go
> <https://github.com/openshift/origin/pull/8484#discussion_r60513048>:
>
> > @@ -209,7 +210,7 @@ func (h *binaryInstantiateHandler) handle(r io.Reader) (runtime.Object, error) {
> >  	if err != nil {
> >  		return nil, errors.NewInternalError(fmt.Errorf("unable to connect to server: %v", err))
> >  	}
> > -	if err := exec.Stream(r, nil, nil, false); err != nil {
> > +	if err := exec.Stream(kubeletremotecommand.SupportedStreamingProtocols, r, nil, nil, false); err != nil {
>
> @ncdc <https://github.com/ncdc> is there a backcompat implication?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/8484/files/acac24a0bac7bfcf9b7a4adefadb89114d76ecb9..8e14778586be49d76a96c218244278b074fa9766#r60513048>
>